source/kernel/src/registry/mod.rs

```diff
@@ -27,13 +27,15 @@ pub mod known_uuids {
         pub const SERIAL_MUX: Uuid = uuid!("54c983fa-736f-4223-b90d-c4360a308647");
         pub const SIMPLE_SERIAL_PORT: Uuid = uuid!("f06aac01-2773-4266-8681-583ffe756554");
         pub const EMB_DISPLAY: Uuid = uuid!("b54db574-3eb7-4c89-8bfb-1a20890be68e");
+        pub const I2C: Uuid = uuid!("011ebd3e-1b14-4bfd-b581-6138239b82f3");
```

Comment:
Doesn't need to be fixed in this PR, but we *probably* want to think about including date/version values here. Eventually we should care about breaking interface changes.

Reply:
yup, good idea!